velox/common/base/BloomFilter.h

```diff
@@ -19,21 +19,35 @@
 #include <cstdint>
 #include <vector>
 
-#include <folly/Hash.h>
-
 #include "velox/common/base/BitUtil.h"
+#include "velox/common/base/Exceptions.h"
+#include "velox/common/base/IOUtils.h"
 
 namespace facebook::velox {
+namespace {
+const int8_t kBloomFilterV1 = 1;
+common::InputByteStream initializeInputStream(const char* serialized) {
```

Comment:
One more small thing, `inline` since this is in header.  Also do not use anonymous namespace in header file, you can use something like `detail` or `impl` or `internal`